doc/api/apis/mappings.raml

```diff
@@ -1,6 +1,7 @@
 #type: collection
 get:
   is: [ embeddable: { embedFields: "user,map" }, orderable, pageable ]
+  securedBy: [ null, cookie, token, oauth_2_0 ]
```

Comment:
these ones say cookie, token, oauth2, and I think it should be the other way around?